app/components/autocomplete/slash_suggestion/non_cached/slash_suggestion.js

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {
+    FlatList,
+    Platform,
+} from 'react-native';
+
+import AutocompleteDivider from 'app/components/autocomplete/autocomplete_divider';
+import {makeStyleSheetFromTheme} from 'app/utils/theme';
+
+import SlashSuggestionItem from '../slash_suggestion_item';
+import {Client4} from '@mm-redux/client';
+import {COMMANDS_TO_HIDE_ON_MOBILE} from '../cached/index';
+
+export default class SlashSuggestionNonCached extends PureComponent {
```

Comment:
Note that I did not create a parent class here therefore `SlashSuggestionNonCached` and `SlashSuggestionCached` have some repeated code. Composition will not work, not sure if we should use inheritance.